packages/testing-utils/src/index.js

```diff
@@ -18,20 +31,54 @@ export const cloneableGenerator = generatorFunc => (...args) => {
   }
 }
 
+const assertStatusRunning = status => {
+  if (status !== RUNNING) {
+    const str = statusToStringMap[status]
+    throw new Error(
+      `The task is no longer Running, it is ${str}. You can't change the status of a task once it is no longer running.`,
+    )
+  }
+}
+
 export function createMockTask() {
-  let _isRunning = true
-  let _result
-  let _error
+  let status = RUNNING
```

Comment:
nit: in general, it feels that keeping this as a number doesn't buy us anything here since we have to remap this later to string anyway when throwing an error. We could just ditch that if we'd use string status here.

I don't feel strongly about this though - both approaches are fine.